tests/integration/entity/entity.spec.ts

```diff
@@ -159,6 +159,16 @@ describe('entity', function () {
         expect(response.body).toMatchObject(entity);
       });
 
+      it('should return 200 status code and only the osmId', async function () {
```

Comment:
missing in description "with header of ....."

Reply:
Done